src/main/java/eu/arrowhead/blacklist/service/ManagementService.java

```diff
@@ -0,0 +1,135 @@
+	//-------------------------------------------------------------------------------------------------
+	public BlacklistEntryListResponseDTO create(final BlacklistCreateListRequestDTO dto, final String origin, final String requesterName) {
+		logger.debug("ManagementService create started...");
+
+		final BlacklistCreateListRequestDTO normalizedDto = validator.validateAndNormalizeBlacklistCreateListRequestDTO(dto, origin);
+		final String normalizedRequesterName = validator.validateAndNormalizeSystemName(requesterName, origin);
+		checkSelfBlacklisting(normalizedDto.entities(), normalizedRequesterName, origin);
+		whitelistService.checkWhitelist(normalizedDto.entities().stream().map(e -> e.systemName()).collect(Collectors.toList()), origin);
+		List<Entry> createdEnties;
+		try {
+			createdEnties = dbService.createBulk(normalizedDto.entities(), normalizedRequesterName);
+		} catch (final InternalServerError ex) {
+			throw new InternalServerError(ex.getMessage(), origin);
+		}
+		return dtoConverter.convertEntriesToBlacklistEntryListResponseDTO(createdEnties, createdEnties.size());
+	}
+
+	//-------------------------------------------------------------------------------------------------
+	public void remove(final List<String> systemNameList, final boolean isSysop, final String revokerName, final String origin) {
+		logger.debug("ManagementService remove started...");
+
+		// only sysop can remove istelf from blacklist
+		checkSysopRemoval(systemNameList, isSysop, origin);
+
+		final List<String> normalizedList = validator.validateAndNormalizeSystemNameList(systemNameList, origin);
+		final String normalizedRevokerName = validator.validateAndNormalizeSystemName(revokerName, origin);
+		try {
+			dbService.unsetActiveByNameList(normalizedList, normalizedRevokerName, origin);
+		} catch (final InternalServerError ex) {
+			throw new InternalServerError(ex.getMessage(), origin);
+		}
+	}
+
+	//=================================================================================================
+	// assistant methods
+
+	//-------------------------------------------------------------------------------------------------
+	private void checkSelfBlacklisting(final List<BlacklistCreateRequestDTO> candidates, final String requesterName, final String origin) {
+		Assert.notNull(candidates, "Candidates is null!");
+		Assert.isTrue(!Utilities.containsNull(candidates), "Candidate list contains null element!");
+
+		for (final BlacklistCreateRequestDTO candidate : candidates) {
+			if (candidate.systemName().equals(requesterName)) {
+				throw new InvalidParameterException("It is not allowed to add yourself to the blacklist!", origin);
+			}
+		}
+	}
+
+	//-------------------------------------------------------------------------------------------------
+	private void checkSysopRemoval(final List<String> names, final boolean isSysop, final String origin) {
+		Assert.notNull(names, "System name list is null!");
+		if (names.contains(Constants.SYSOP) && !isSysop) {
```

Comment:
Being a sysop does not mean your name is sysop (only in self declared mode). You have to add the requesterName as parameter too. And if requesterName is in the list of names and isSysop is false, then you have to throw the exception